test/check/check.go

```diff
@@ -649,6 +658,21 @@ func checkForPrometheusRule(name string) error {
 	return nil
 }
 
+func checkForNetworkAttachmentDefinition(name string) error {
+	networkAttachmentDefinition := k8snetworkplumbingwgv1.NetworkAttachmentDefinition{}
+	err := testenv.Client.Get(context.Background(), types.NamespacedName{Name: name, Namespace: corev1.NamespaceDefault}, &networkAttachmentDefinition)
+	if err != nil {
+		return err
+	}
+
+	err = checkRelationshipLabels(networkAttachmentDefinition.GetLabels(), "NetworkAttachmentDefinition", name)
+	if err != nil {
+		return err
+	}
+
+	return nil
```

Comment:
here you can just return err basically, but one can consider it less readable
so either this please or short writing 
```
if ...; err != nil {
}
```

Reply:
[same](https://github.com/kubevirt/cluster-network-addons-operator/pull/1841/commits/f634d35bfb3cee4337321935d578c78af7f5d6cf#r1708736686)